docs/guides/model-your-first-process.md

```diff
@@ -16,24 +16,30 @@ import SaasPrereqs from './react-components/\_saas-prerequisites.md'
 
 ## Design and deploy a process
 
-In Camunda 8, you have two options to design and deploy a process, but for the purposes for this guide you will find instructions for [Web Modeler](../components/modeler/about-modeler.md).
+With Camunda 8, you can design and deploy a process using both [Desktop Modeler and Web Modeler](/components/modeler/about-modeler.md). For this guide, we will use [Web Modeler](../components/modeler/about-modeler.md).
 
-1. From Modeler, click **New project**.
+1. From Web Modeler, select **Create new project**.
 2. Name your project and select **Create new > BPMN diagram**.
    ![blank project create bpmn diagram](./img/blank-project.png)
-3. Give your model a descriptive name, and then give your model a descriptive ID within the **General** tab inside the properties panel on the right side of the screen.
-4. Create a task by dragging the rectangular task icon from the palette, or by clicking the existing start event and clicking the task icon. Make sure there is an arrow connecting the start event to the task.
+3. Give your model a descriptive name and ID within the **General** tab inside the properties panel on the right side of the screen. For example, you may name the diagram `Process invoice` with an ID of `process_invoice`.
```

Comment:
As a first time user, I would wonder what the ID means/does -- we should provide guidance on this.